zigpy_znp/types/named.py

```diff
@@ -186,6 +186,8 @@ class Status(MissingEnumMixin, basic.enum_uint8):
 
     MALFORMED_CMD = 0x80
     UNSUP_CLUSTER_CMD = 0x81
+    # The requested optional feature is not supported on the target device.
+    NOT_SUPPORTED = 0x84
```

Comment:
You can get rid of this, since it's already present as `t.ZDOStatus.NOT_SUPPORTED`